packages/ckeditor5-table/tests/_utils/utils.js

```diff
@@ -32,20 +32,26 @@ const WIDGET_TABLE_CELL_CLASS = 'ck-editor__editable ck-editor__nested-editable'
  *		};
  *
  * @param {Array.<Array.<String>|Object>} tableData
- * @param {Object} [attributes] Optional table attributes: `headingRows` and `headingColumns`.
+ * @param {Object} [options={}] Optional table attributes. Supports {@link module:table/table~TableConfig#tableProperties}.
+ * @param {Number} options.headingRows Number of heading rows.
+ * @param {Number} options.headingColumns Number of heading columns.
+ * @param {module:table/table~TableConfig#tableCellProperties} options.cellProperties Properties for all created cells.
  *
  * @returns {String}
  */
-export function modelTable( tableData, attributes ) {
+export function modelTable( tableData, options = {} ) {
 	const tableRows = makeRows( tableData, {
 		cellElement: 'tableCell',
 		rowElement: 'tableRow',
 		headingElement: 'tableCell',
 		wrappingElement: 'paragraph',
-		enforceWrapping: true
+		enforceWrapping: true,
+		cellProperties: options.cellProperties
 	} );
 
-	return `<table${ formatAttributes( attributes ) }>${ tableRows }</table>`;
+	delete options.cellProperties;
+
+	return `<table${ formatAttributes( options ) }>${ tableRows }</table>`;
 }
```

Comment:
I don't like the idea of mixing the table element attributes with the `cellProperties` meta field. Maybe we could leave the `attributes` as it was and introduce additional `options` parameter?